src/hole_punching.rs

```diff
@@ -220,12 +235,12 @@ impl HolePunchServer {
                     Ok(())  => return Ok(()),
                 }
                 let mut data_recv = [0u8; MAX_DATAGRAM_SIZE];
-                /*
-                 * TODO (canndrew):
-                 * Currently we set a read timeout so that the hole punch server thread continually wakes
-                 * and checks to see if it's time to exit. This is a really crappy way of implementing
-                 * this but currently rust doesn't have a good cross-platform select/epoll interface.
-                 */
+        //
+        // TODO (canndrew):
+        // Currently we set a read timeout so that the hole punch server thread continually wakes
+        // and checks to see if it's time to exit. This is a really crappy way of implementing
+        // this but currently rust doesn't have a good cross-platform select/epoll interface.
+        //
```

Comment:
I can understand the switch from '/* ... */' to just '//', but I don't understand the rule for indentation, why do we indent minus 8 spaces?